### ISSUE TITLE
Is this accurate? wait time question not showing anything in iOS dark mode

### ISSUE THREAD

**Issue by kungpaogao** [cornell-dti/campus-density-ios#52]
## Overview
Nothing shows for "How many minutes did you wait in line?" when iPhone is in dark mode. User can still scroll through the selection and feel haptic feedback. This impacts a user's ability to report accurate information.

## Fixes
- more dark mode testing
- probably some iOS theming fixes

## Screenshot
<img width="414" alt="image" src="https://user-images.githubusercontent.com/11168401/117610186-4f7b6c00-b12f-11eb-8484-bf3c184955b0.png">

**Comment by Coreball** [comment]
Good catch. I just noticed that the text input for the search bar on the main view is black by default in dark mode too.
